packages/collector/test/tracing/database/ioredis/test.js

```diff
@@ -42,8 +43,56 @@ function checkConnection(span, setupType) {
 //    export AZURE_REDIS_CLUSTER=team-nodejs-redis-cluster-tekton.redis.cache.windows.net:6380
 //    export AZURE_REDIS_CLUSTER_PWD=
 ['default', 'cluster'].forEach(setupType => {
+  if (setupType !== 'cluster') {
```

Comment:
Why the tests skip case for cluster?

Reply:
@kirrg001 
could you please comment on this ?

Reply:
Because the `allowRootExitSpanApp` has no implementation for cluster. 
@abhilash-sivan Please add a comment

Reply:
working on it

Reply:
@abhilash-sivan We only need a comment.

Reply:
I will create a new PR to implement the allowRootExitSpanApp in cluster mode.

Reply:
No. Please track this as a TODO on the PR for now. We will raise a card.